docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/configuring-product-visibility-by-site-using-channels.md

```diff
@@ -1,6 +1,6 @@
-# Configuring Product Visibility by Site Using Channels
+# Configuring Product Visibility by Channel
 
-By default in Liferay Commerce Products added to any catalog are visible globally - products are visible across all sites added to a Liferay Commerce installation. Store administrators may want to limit product visibility to particular channels based on their criteria.
+By default, Products are visible across all channels, but visibility can be limited to specific channels.
 
 For the purpose of this example, we will consider a scenario of a multi-region seller providing products from a single catalog.
```

Comment:
1) the diagram below is inaccurate--the filtering setting is shared across the entire product, not on a channel-basis.
2) the diagram below shows too many layers for the concept it's trying to convey. Catalogs are irrelevant in this context, as is the related site to the channel. I'd suggest simplifying so it's showing just the relationship between a product and channels.
3) the diagram has too much going on which distracts from the core idea. the core ideas in my mind are a product can be visible in all channels or filtered for some channels. There are three channels but the setting for Car Store NA and Car Store EU are identical, so why include both channels? I'd suggest removing one of them.
4) I've tended to avoid the multi-region example when explaining channels just so we avoid making any implication that locales are tightly coupled with channels. To avoid this, I've been using the multi-brand scenario, where the same products are being shared across multiple brands and each site has a unique brand experience. I suggest making the same edit here.